cli/getting-started-cli.adoc

```diff
@@ -30,15 +30,39 @@ include::modules/install-roxctl-cli-windows.adoc[leveloffset=+3]
 include::modules/run-roxctl-from-container.adoc[leveloffset=+2]
 
 // Authenticating using CLI
+[id="authenticating-using-the-roxctl-cli_{context}"]
+== Authenticating using the `roxctl` CLI
 
+For authentication, you can use an API token, your administrator password, or `roxctl login` command.
+
+* Use an API token in a production environment within Continous Intergration (CI) or automation because each token is assigned specific access permissions.
```

Comment:
I suppose we could also mention that beside it being assigned to a specific access permission, it doesn't require any interactive flows, i.e. no user needs to login with browser etc. and has a TTL of 1 year.